packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionModule.tests.js

```diff
@@ -142,7 +142,9 @@ QUnit.module('Mention format', () => {
         assert.strictEqual(element.dataset.marker, '@', 'correct marker');
         assert.strictEqual(element.dataset.mentionValue, 'John Smith', 'correct value');
         assert.strictEqual(element.dataset.id, 'JohnSm', 'correct id');
-        assert.strictEqual(element.innerText, '@John Smith', 'correct inner text');
+
+        new MentionFormat({}, element);
+        assert.strictEqual(element.children[0].innerText, '@John Smith', 'correct inner text');
```

Comment:
[atomic test]
Now this test checks 2 different points:
1. ` MentionFormat.create(data);` creates a node with passed data attributes
2. ` new MentionFormat({}, element);` renders content getting the attributes from passed node

I suggest you to split this tests to make it atomic and more clear.